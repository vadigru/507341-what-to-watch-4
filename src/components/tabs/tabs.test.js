import React from "react";
import renderer from "react-test-renderer";
import Tabs from "../tabs/tabs.jsx";

const mock = {
  activeTab: `Overview`,
  tabNames: [`Overview`, `Details`, `Reviews`],
};

it(`Should render Tabs component`, () => {
  const tree = renderer
    .create(
        <Tabs
          tabNames={mock.tabNames}
          activeTab={mock.activeTab}
          onTabClick={() => {}}
          onGenreTabClick={() => {}}
        />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
