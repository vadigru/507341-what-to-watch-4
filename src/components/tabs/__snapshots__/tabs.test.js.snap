// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Tabs component 1`] = `
Array [
  <li
    className="movie-nav__item movie-nav__item--active"
  >
    <a
      className="movie-nav__link"
      href="#"
    >
      Overview
    </a>
  </li>,
  <li
    className="movie-nav__item "
  >
    <a
      className="movie-nav__link"
      href="#"
    >
      Details
    </a>
  </li>,
  <li
    className="movie-nav__item "
  >
    <a
      className="movie-nav__link"
      href="#"
    >
      Reviews
    </a>
  </li>,
]
`;
